README.md

```diff
@@ -98,10 +98,10 @@ it executes the following algorithm:
 1. Looks for the `ServiceCatalogEntry` 3PR in the catalog
   - If not found, sets the `status` field to `Failed`, adds an appropriate explanation to the `statusDescription` field
     field to a human-readable description of the error, and stops processing
-1. Looks in the `action` field of the claim and takes the appropriate action
+2. Looks in the `action` field of the claim and takes the appropriate action
   - Valid values are `provision`, `bind`, `unbind`, `deprovision`, `create` and `delete`. See [`ServicePlanClaim` documentation](./doc/DATA_STRUCTURES.md#serviceplanclaim) for details on each value
```

Comment:
@rcaron can you add 2 more spaces before this and the below `-` chars? I'm not certain, but I believe that should indent them properly